src/aggregators/aggregate.js

```diff
@@ -20,6 +20,28 @@
 class Aggregate {
 
   /**
+   * The valid types of aggregation supported by Lighthouse.
+   */
+  static get VALID_TYPES() {
+    return {
+      PWA: {
+        name: 'Progressive Web App',
+        contributesToScore: true,
+        report: {
+          tag: false
```

Comment:
ah, I see. Yeah, that seems like a weird flag to put on the aggregation type, but your call on what's cleaner